app/qml/account/MMAccountController.qml

```diff
@@ -46,24 +46,6 @@ Item {
     stackView.pop(null);
   }
 
-  function shuffleAndAppend( listModel, items )
```

Comment:
We can keep shuffleAndAppend method exclusively in MMAccountController, as it will be inherited by both MMWhichIndustryPage and MMHowYouFoundUsPage. This way, there is no need to declare it twice.

Reply:
It was moved to make the "MerginMapsGallery" app working as the gallery don't inherit MMAccountController, afaik but maybe there is a more clever way to separate business logic with UI